add local eval flag configs api and bootstrapping docs (#663)
amplitude/amplitude-docs: content/collections/experiment-sdks/en/experiment-android.md

```swift
let config = ExperimentConfigBuilder()
    .initialVariants(["<FLAG_KEY>": Variant("<VARIANT>")])
    .source(Source.InitialVariants)
    .build()
let experiment = Experiment.initialize(apiKey: "<DEPLOYMENT_KEY>", config: config)
```

{{partial:tabs tabs="Java, Kotlin"}}
{{partial:tab name="Java"}}
```java
ExperimentConfig config = ExperimentConfig.builder()
    .initialVariants(Map.of("<FLAG_KEY>", new Variant("<VARIANT>")))
    .source(Source.INITIAL_VARIANTS)
    .build();
ExperimentClient experiment = Experiment.initialize(
    context, "<DEPLOYMENT_KEY>", config);
```
{{/partial:tab}}
{{partial:tab name="Kotlin"}}
```kotlin
val config = ExperimentConfig.builder()
    .initialVariants(mapOf("<FLAG_KEY>" to Variant("<VARIANT>")))
    .source(Source.INITIAL_VARIANTS)
    .build()
val experiment = Experiment.initialize(context, "<DEPLOYMENT_KEY>", config)
```
{{/partial:tab}}
{{/partial:tabs}}

### Bootstrapping flag configurations

> You may choose to bootstrap the SDK with an initial set of local evaluation flag configurations using the `initialFlags` configuration. These will be evaluated  when [variant](#variant) is called, unless an updated flag config or variant is loaded with [start](#start) or [fetch](#fetch).

Reviewer: ```suggestion
You may choose to bootstrap the SDK with an initial set of local evaluation flag configurations using the `initialFlags` configuration. Experiment evaluates these when you call [variant](#variant), unless you load an updated flag config or variant with [start](#start) or [fetch](#fetch).
```
